tx-pool/src/component/mod.rs

```diff
@@ -20,7 +20,7 @@ const DEFAULT_BYTES_PER_CYCLES: f64 = 0.000_170_571_4_f64;
 
 /// Virtual bytes(aka vbytes) is a concept to unify the size and cycles of a transaction,
 /// tx_pool use vbytes to estimate transaction fee rate.
-pub(crate) fn get_transaction_virtual_bytes(tx_size: usize, cycles: u64) -> u64 {
```

Comment:
Could this function moved to `ckb-types`?

It's too heavy for third-party applications to call this function, they have to depend on `rocksdb`, `tokio` and so on.

Reply:
It's fine to expose it. I didn't want to expose this function, it's so simple that there's no need to import it from `ckb-types`, other non-rust projects can't use it either.

Reply:
If expose it, at least, users could find several documentation lines.
It's really hard to find it.
Last week, a former ckb-core member had asked me for how to calculate `vbytes`.

If I was correct, there is no RFC mention there is a concept of "vbytes" in CKB and the magic number `DEFAULT_BYTES_PER_CYCLES`.
From normal users' point, they may not know there is a concept of "vbyes" in CKB.

And, no standard (RFC) means there is no promise whether the number `DEFAULT_BYTES_PER_CYCLES` would be changed, so depend on an official API is more safe than hard-code.

**Maybe we should standardize this concept, better than force users to read our code to find the magic number and the cryptic formula.**

### References

- [Section "Paying for Transaction Fees" in CKB RFC-0015](https://github.com/nervosnetwork/rfcs/blob/af5c8f32ef6142abe520c6e6cc688049db125f08/rfcs/0015-ckb-cryptoeconomics/0015-ckb-cryptoeconomics.md#paying-for-transaction-fees)
- [Section "Computation Cost and Transaction Fees" in CKB RFC-0002](https://github.com/nervosnetwork/rfcs/blob/af5c8f32ef6142abe520c6e6cc688049db125f08/rfcs/0002-ckb/0002-ckb.md#52-computation-cost-and-transaction-fees)